common/service/apml/apml.h

```diff
@@ -142,6 +144,8 @@ void apml_init();
 void fatal_error_happened();
 void apml_recovery();
 int pal_check_sbrmi_command_code_length();
+uint8_t plat_set_apml_bus();
+uint8_t get_apml_bus();
```

Comment:
Better to name this apml_get_bus() to bring it in sync with other APIs with name prefix as apml_*

Reply:
Modify the function name to apml_get_bus().